src/einsteinpy/metric/kerrnewman.py

```diff
@@ -102,15 +103,15 @@ def metric_covariant(self, x_vec):
 
         Raises
         ------
-        NotImplementedError
+        CoordinateError
             In case of the metric is not available in \
```

Comment:
👀

Reply:
Just look these errors up across the module and replace with the aforementioned lines.